packages/client/src/products/__tests__/getRecommendedSet.test.ts

```diff
@@ -5,31 +5,21 @@ import {
 } from 'tests/__fixtures__/products';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getRecommendedSet.fixtures';
-import moxios from 'moxios';
+import mswServer from '../../../tests/mswServer';
 
 describe('recommended sets client', () => {
   const expectedConfig = undefined;
 
-  beforeEach(() => {
-    moxios.install(client);
-    jest.clearAllMocks();
-  });
-
-  afterEach(() => moxios.uninstall(client));
+  beforeEach(jest.clearAllMocks);
 
   describe('getRecommendedSet()', () => {
     const spy = jest.spyOn(client, 'get');
 
     it('should handle a client request successfully', async () => {
-      const response = mockRecommendedSet;
-
-      fixtures.success({
-        id: mockRecommendedSetId,
-        response,
-      });
+      mswServer.use(fixtures.success(mockRecommendedSet));
 
```

Comment:
This would look great with `expect.assertions(2);` here :)

Reply:
Added 🙂